release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -79,7 +79,191 @@ For more information, see xref:../openshift_images/image-streams-manage.adoc#ima
+* xref:../installing/installing_ibm_z/installing-restricted-networks-ibm-z-kvm.adoc#installing-restricted-networks-ibm-z-kvm[Installing a cluster with RHEL KVM on IBM Z and LinuxONE in a restricted network]
+
+[discrete]
+==== Notable enhancements
+
+The following new features are supported on {ibmzProductName} and {linuxoneProductName} with {product-title} {product-version}:
+
+* Assisted Installer
+* Cluster Resource Override Operator
+* Egress IP
+* MetalLB Operator
+* Network-Bound Disk Encryption - External Tang Server
+
+[discrete]
+==== IBM Secure Execution
+
+{product-title} now supports configuring {op-system-first} nodes for IBM Secure Execution on {ibmzProductName} and {linuxoneProductName} (s390x architecture).
+
+For installation instructions, see the following documentation:
+
+* xref:../installing/installing_ibm_z/installing-ibm-z-kvm.html#installing-rhcos-using-ibm-secure-execution_installing-ibm-z-kvm[Installing {op-system} using IBM Secure Execution]
+
+[discrete]
+==== Supported features
+
+The following features are also supported on {ibmzProductName} and {linuxoneProductName}:
+
+* Currently, the following Operators are supported:
+** Cluster Logging Operator
+** Compliance Operator
+** File Integrity Operator
+** Local Storage Operator
+** NFD Operator
+** NMState Operator
+** OpenShift Elasticsearch Operator
+** Service Binding Operator
+** Vertical Pod Autoscaler Operator
+* The following Multus CNI plugins are supported:
+** Bridge
+** Host-device
+** IPAM
+** IPVLAN
+* Alternate authentication providers
+* Automatic Device Discovery with Local Storage Operator
+* CSI Volumes
+** Cloning
+** Expansion
+** Snapshot
+* Cron jobs
+* Descheduler
+* Encrypting data stored in etcd
+* Helm
+* Horizontal pod autoscaling
+* IPv6
+* Monitoring for user-defined projects
+* Multipathing
+* Operator API
+* OC CLI plugins
+* Persistent storage using iSCSI
+* Persistent storage using local volumes (Local Storage Operator)
+* Persistent storage using hostPath
+* Persistent storage using Fibre Channel
+* Persistent storage using Raw Block
+* PodDisruptionBudget
+* OVN-Kubernetes, including IPsec encryption
+* Scheduler profiles
+* Stream Control Transmission Protocol (SCTP)
+* Support for multiple network interfaces
+* Three-node cluster support
+* z/VM Emulated FBA devices on SCSI disks
+* 4K FCP block device
+
+These features are available only for {product-title} on {ibmzProductName} and {linuxoneProductName} for {product-version}:
+
+* HyperPAV enabled on {ibmzProductName} and {linuxoneProductName} for the virtual machines for FICON attached ECKD storage
+
+[discrete]
+==== Restrictions
+
+The following restrictions impact {product-title} on {ibmzProductName} and {linuxoneProductName}:
+
+* Automatic repair of damaged machines with machine health checking
+* {openshift-local-productname}
+* Controlling overcommit and managing container density on nodes
+* NVMe
+* OpenShift Metering
```

Comment:
```suggestion

```
Support for Metering has been removed. We don't need to call out the restriction anymore.

Reply:
Ack.